src/main/webapp/app/registry/encryption/encryption.component.html

```diff
@@ -0,0 +1,66 @@
+<div>
+    <div class="head">
+        <h2 class="d-inline-block">Encrypt and decrypt properties</h2>
+    </div>
+    <div class="panel panel-default">
+        <div class="panel-body">
+            <ngb-tabset type="pills" class="nav-fill">
```

Comment:
pills
![screen shot 2017-06-26 at 18 44 01](https://user-images.githubusercontent.com/513471/27550311-800423fc-5a9f-11e7-867f-570eebaeb81b.png)
tabs
![screen shot 2017-06-26 at 18 44 15](https://user-images.githubusercontent.com/513471/27550312-8006346c-5a9f-11e7-853e-a7175ede069a.png)

I personally prefer pills as I don't know how to fix the styling for tabs...